### DISCUSSION TITLE
semantic error TS2322: Type 'string' is not assignable to type 'never'

### DISCUSSION THREAD

**Discussion by biotech7** [epam/ketcher#1516] [General]
when building ketcher-react module, it returned an error in _create-react-app_ project:
**semantic error TS2322: Type 'string' is not assignable to type 'never'.**
**Error: ...../ketcher-react/src/script/ui/App/App.tsx(65,17):** 

is this a bug or only imcompatible with _create-react-app_?

**Comment by AndreiMazol** [comment]
Dear @biotech7 ,

ketcher-react package can be integrated into CRA without any issues. For example of usage you can look at https://github.com/epam/ketcher/tree/master/example or https://github.com/epam/ketcher/tree/master/demo. 

Please provide more details about case:
1. ketcher-react version 
2. CRA version
3.  if it is possible please provide snapshot which represents integration ketcher into your project.

Best regards,
Andrei
